opensrp-anc/src/test/java/org/smartregister/anc/library/activity/BaseActivityUnitTest.java

```diff
@@ -69,7 +69,6 @@ public void setUp() {
         Mockito.when(configurableViewsLibrary.getConfigurableViewsHelper()).thenReturn(configurableViewsHelper);
 
         ReflectionHelpers.setStaticField(ConfigurableViewsLibrary.class, "instance", configurableViewsLibrary);
-        ConfigurableViewsHelper configurableViewsHelper = Mockito.mock(ConfigurableViewsHelper.class);
```

Comment:
Any reason we are deleting this mock?

Reply:
This wasn't used anywhere and code cleaner removed it.